scripts/murmur.ini

```diff
@@ -139,9 +139,23 @@ serverpassword=
 ; to send speech at.
 bandwidth=72000
 
+; Murmur and Mumble are usually pretty good about cleaning up hung clients, but
+; occasionally one will get stuck on the server. The timeout setting will cause
+; a periodic check of all clients who haven't communicated with the server in
+; this many seconds - causing zombie clients to be disconnected.
+;
+; Note that this has no effect on idle clients or people who AFK. It will only
```

Comment:
I think the phrasing should be "who **are** AFK"

Reply:
Done